src/components/StoreDisplay/StoreBasicInfo/StoreBasicInfo.tsx

```diff
@@ -8,48 +8,76 @@ import {
   KeywordBox,
   StarPoint,
 } from './StoreBasicInfo.styles'
+import { getBrandImg } from '@services/markerImg'
+import { useSelector } from 'react-redux'
+import { RootState } from '@stores/store'
+import KeywordBadge from '@styles/KeywordBadge'
+
+interface StoreInfo {
+  storeImg: string
+  place_name: string
+  address_name: string
+  phone: string
+  keywordList: string[]
+  starCount: number
+}
 
 const StoreBasicInfo = () => {
+  const imgRef = useRef<HTMLImageElement | null>(null)
+  const selectedStore = useSelector(
+    (state: RootState) => state.conv.selectedStore
+  )
+  const [storeInfo, setStoreInfo] = useState<StoreInfo>({
+    storeImg: funlogImg,
+    place_name: 'FUN편로그 편의점',
+    phone: '  02-525-2525',
+    address_name: ' 서울시 어쩌구 무슨무슨로 2-1',
+    starCount: 3,
+    keywordList: ['제품이 다양해요', '매장이 청결해요', '펀편로그 좋아요'],
+  })
+
+  useLayoutEffect(() => {
+    if (selectedStore) {
+      const [placeName] = selectedStore.place_name.split(' ')
+      setStoreInfo((prevState) => {
+        return {
+          prevState,
+          ...selectedStore,
+          storeImg: getBrandImg(placeName) ?? funlogImg,
+        }
+      })
+    }
+  }, [selectedStore])
+
   return (
     <InfoContainer>
       <ConvImgWrapper>
-        <img src={GsImg} />
+        <img src={storeInfo.storeImg} ref={imgRef} />
       </ConvImgWrapper>
 
       <ConvInfo>
-        <h1>GS25 땡땡점</h1>
+        <h1>{storeInfo.place_name}</h1>
         <p>
           <span>
-            <PushpinFilled /> 서울시 어쩌구 무슨무슨로 2-1
+            <PushpinFilled />
+            {storeInfo.address_name}
           </span>
           <span>
             <PhoneFilled />
-            02-525-2525
+            {storeInfo.phone}
           </span>
         </p>
         <KeywordBox>
           <ul>
-            <li>
-              <span>제품이 다양해요</span>
-            </li>
-            <li>
-              <span>제품이 다양해요</span>
-            </li>
-            <li>
-              <span>제품이 다양해요</span>
-            </li>
-            <li>
-              <span>제품이 다양해요</span>
-            </li>
-            <li>
-              <span>제품이 다양해요</span>
-            </li>
+            {storeInfo.keywordList.slice(0, 5).map((keyword) => {
+              return <KeywordBadge key={keyword}>{keyword}</KeywordBadge>
+            })}

```

Comment:
(p4) Return 을 축약을 하는건 어떨까 합니다

```suggestion
            {storeInfo.keywordList.slice(0, 5).map((keyword) => (
              <KeywordBadge key={keyword}>{keyword}</KeywordBadge>
            ))}
```